### DISCUSSION TITLE
Include OIDC claims in github context

### DISCUSSION THREAD

**Discussion by wlynch** [community/community#34235] [Actions]
It would be great if the [`github` context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) could include all of the claims included in [Actions OIDC tokens](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect)

Values that are currently missing:

- sub
- environment
- actor_id / repo_id / repo_owner_id 
- job_workflow_ref
- repository_visibility

**Comment by GF-Elite** [comment]
...

**Comment by feelepxyz** [comment]
@wlynch the actions team recently added the following ENV on the hosted runners:
- `GITHUB_ACTOR_ID`
- `GITHUB_REPOSITORY_ID`
- `GITHUB_REPOSITORY_OWNER_ID`

There's also `WORKFLOW_REF` and `WORKFLOW_SHA` that point to the triggering/initiating workflow. `job_workflow_ref` did not get added as part of this. 

Repo visibility is in the event payload, e.g.
```
const event = JSON.parse(fs.readFileSync(process.env["GITHUB_EVENT_PATH"]));
console.log(JSON.stringify(event, null, 2));
const isPrivate  = event.repository.private
```

Not sure `sub` makes sense as this seems specific to the oidc token and can be customised when minting the token?
